packages/block-library/src/image/edit.native.js

```diff
@@ -564,8 +561,21 @@ export class ImageEdit extends Component {
 				<PanelBody title={ __( 'Link Settings' ) }>
 					{ this.getLinkSettings( true ) }
 				</PanelBody>
-				{ canImageBeFeatured &&
-					this.getFeaturedButtonPanel( isFeaturedImage ) }
+				<PanelBody
+					title={ __( 'Featured Image' ) }
+					titleStyle={ styles.featuredImagePanelTitle }
+				>
+					{ canImageBeFeatured &&
+						this.getSetFeaturedButton( isFeaturedImage ) }
```

Comment:
I tested these changes but I'm getting an error that states that the function `getSetFeaturedButton` doesn't exist, looks like the function name has been changed and in fact, doesn't exist, however, I see that `getFeaturedButtonPanel` is present. 

I'm wondering if this should be calling `getFeaturedButtonPanel` instead of `getSetFeaturedButton`, wdyt?

Reply:
Ah, yes, thank you for spotting this! The function was updated to `getFeaturedButtonPanel` following feedback on the iOS side, which was merged recently. A merge error on my part is the reason the old name is being referenced here. Fixed in https://github.com/WordPress/gutenberg/pull/33057/commits/658d0054501f589402b674f006caec38aadc893c.